stracer-circe/src/test/resources/com/evolutiongaming/stracer/trace.json

```diff
@@ -1,6 +1,7 @@
 {
   "traceId": "5c7ef48d0000303900000000075bcd15",
   "spanId": "000001694f93b479",
+  "parentId": null,
```

Comment:
why null ?

Reply:
Circe generates this json for `None` values.